cdk-testsuite/libraries/openshift/openshift.py

```diff
@@ -0,0 +1,251 @@
+'''
+Created on Jun 29, 2016
+
+@author: amit
+'''
+from avocado.utils import process
+import imp
+import logging
+import re
+import time
+
+log = logging.getLogger("Openshift.Debug")
+
+
+def openshiftLibInfo(self):
+    '''
+    Get openshift user defined library version
+    Args:
+        self (object): Object of the current method
+    '''
+    openshiftUtils = imp.load_source('openshiftUtils', self.params.get('openshift_util_MODULE'))
+    self.log.info("Openshift library version : " +openshiftUtils.get_version())
+    
+def oc_usr_login(self, ip_port, uname, password):

```

Comment:
I would like to have ip_port as optional argument, with default value for localhost and default port.